fix: link the readme for setup (#768)
openfga/openfga.dev: docs/content/getting-started/setup-openfga/configure-openfga.mdx

# Configuring OpenFGA

> Before configuring OpenFGA, follow [Getting Started](https://github.com/openfga/openfga?tab=readme-ov-file#getting-started) to set up and install openfga. You can then run `./openfga run` as a pre-compiled binary or using your preferred method.

Reviewer: ```suggestion
Refer to the [OpenFGA Getting Started](https://github.com/openfga/openfga?tab=readme-ov-file#getting-started) for info on the various ways to install OpenFGA.

The instructions below assume OpenFGA is installed and that you have the `openfga` binary in your PATH. If you have built `openfga` as a binary, but not in your path, you can refer to it directly (e.g. replace `openfga` in the instructions below with `./openfga` or `/path/to/openfga`).
```

Author: Where do we provide instructions on adding `openfga` to your path? It's not in the README I'm linking

Reviewer: When you do brew install it does it automatically

Author: In that case, should I also update the readme so that these instructions are consistent?

Command line parameters take precedence over environment variables and options in the configuration file. They are prefixed with `--` , e.g.

```shell

> ./openfga run

Reviewer: ```suggestion
openfga run \
```

### Postgres

```shell

> ./openfga migrate

Reviewer: ```suggestion
openfga migrate \
```

    --datastore-engine postgres \
    --datastore-uri 'postgres://postgres:password@postgres:5432/postgres?sslmode=disable'

> ./openfga run

Reviewer: ```suggestion
openfga run \
```

The connection URI needs to specify the query `parseTime=true`.

```shell

> ./openfga migrate \

Reviewer: ```suggestion
openfga migrate \
```

    --datastore-engine mysql \
    --datastore-uri 'root:secret@tcp(mysql:3306)/openfga?parseTime=true'

> ./openfga run \

Reviewer: ```suggestion
openfga run \
```

Profiling through [`pprof`](https://github.com/google/pprof/blob/main/doc/README.md) can be enabled on the OpenFGA server by providing the `--profiler-enabled` flag. For example:

```sh

> ./openfga run --profiler-enabled

Reviewer: ```suggestion
openfga run --profiler-enabled
```

```

If you need to serve the profiler on a different port than the default `3001`, you can do so by specifying the `--profiler-addr` flag. For example:

```sh

> ./openfga run --profiler-enabled --profiler-addr :3002

Reviewer: ```suggestion
openfga run --profiler-enabled --profiler-addr :3002
```

Various releases of OpenFGA may have experimental features that can be enabled by providing the `--experimentals` flag or the `experimentals` config.

```

> ./openfga run --experimentals="feature1, feature2"

Reviewer: ```suggestion
openfga run --experimentals="feature1, feature2"
```

```
or if you're using environment variables,
```

> ./openfga -e OPENFGA_EXPERIMENTALS="feature1, feature2" run

Reviewer: ```suggestion
openfga -e OPENFGA_EXPERIMENTALS="feature1, feature2" run
```

To set an alternative address, you can provide the `--metrics-addr` flag (`OPENFGA_METRICS_ADDR` environment variable). For example:

```shell

> ./openfga run --metrics-addr=0.0.0.0:2114

Reviewer: ```suggestion
openfga run --metrics-addr=0.0.0.0:2114
```
